clients/venice-client/src/main/java/com/linkedin/venice/fastclient/StatsAvroGenericStoreClient.java

```diff
@@ -226,6 +171,7 @@ private <R> CompletableFuture<R> recordRequestMetrics(
           }
         }
       } else if (requestContext instanceof BatchGetRequestContext) {
+        // BatchGetRequestContext is also the superclass for ComputeRequestContext
```

Comment:
Semantically, you're right. It can be confusing later on. What I can instead do is:
1. Rename `BatchGetRequestContext` to `MultiKeyRequestContext`.
2. Make a new `BatchGetRequestContext` class and make both `BatchGetRequestContext` and `ComputeRequestContext` with empty body for now - and we can evolve each separately if we have to

Reply:
yes, that might be even cleaner. Thanks.